daprdocs/content/en/getting-started/quickstarts/workflow-quickstart.md

```diff
@@ -2064,6 +2064,68 @@ func RequestApprovalActivity(ctx workflow.ActivityContext) (any, error) {
 
 {{< /tabpane >}}
 
+
+## Step 5: Manage Your Workflow
+
+Now that your workflow is running, let's learn how to manage it using the Dapr CLI.
+
+### View Running Workflows
+
+```bash
+# List all workflows
+dapr workflow list --app-id orderprocessing
+
+# You should see output like:
+# INSTANCE ID          WORKFLOW NAME              CREATED              LAST UPDATED         RUNTIME STATUS
+# order-20240312-001   OrderProcessingWorkflow    2024-03-12 10:00:00  2024-03-12 10:00:05  RUNNING
+```
+
+### Check Workflow History
+
+View the detailed execution history of your workflow:
+
+```bash
```

Comment:
None of these commands are going to work on this Quickstart sample. Have you tried these out? We should strive to have samples that will work with the workflow quickstart, even if this is not the complete set.  I would certainly add one where you create a new workflow with the CLI from a new terminal once the  `order-processor` instance is running with the right input commands. This is what I tried to do, but failed.